docs/resources/misc/query-vs-reference-formulations-example.txt

```diff
@@ -0,0 +1,12 @@
+sequenceDiagram
```

Comment:
Is this used somewhere?

Reply:
this is not for me, was already in the resources

Reply:
Hmmmm weird that git then says that this file and the JPEG rendering is added in this PR

Reply:
The change from `dist` to `docs` folder names is the only difference I can see. The rest is exactly the same